inst/include/sampler.h

```diff
@@ -151,6 +151,16 @@ namespace smc {
         long GetNumber(void) const {return N;}
         ///Returns the number of evolution times stored in the history.
         long GetHistoryLength(void) const {return History.size();}
+        ///Returns the current particle set stored in the history.
+        population<Space> GetHistoryPopulation(long n) const {return History[n].GetValues();}
+        ///Returns a reference to the particle set stored in the history.
+        population<Space> & GetHistoryPopulationRefs(long n) {return History[n].GetValues();}
```

Comment:
My inclination would have been for this to be a const function returning a const reference (both for consistency for what that's worth and because I don't /think/ we want this reference to be used to make changes to sampler internals,  but if anyone can see a good reason for that behaviour then do shout...).

Reply:
Yeah I agree, at the moment I can't think of any reason why people would want to change the history in this way.